backend/services/twitter_client.py

```
@@ -52,33 +90,126 @@ def __init__(self):
+                logger.debug("Initializing Twitter client with OAuth 1.0a")
                 auth = tweepy.OAuth1UserHandler(
-                    self.api_key,
-                    self.api_secret,
-                    self.access_token,
-                    self.access_token_secret,
+                    api_key,
+                    api_secret,
+                    access_token,
+                    access_token_secret,
                 )
                 api = tweepy.API(auth, wait_on_rate_limit=True)
```

Ruff (0.14.5)

Local variable `api` is assigned to but never used

Remove assignment to unused variable `api`

(F841)